pyatlan/model/fluent_search.py

```diff
@@ -350,8 +374,8 @@ def to_request(self) -> IndexSearchRequest:
             dsl.size = self._page_size
         if self.sorts:
             dsl.sort = self.sorts
-        # if self.aggregations:
-        #     dsl.aggregations.extend(self.aggregations)
+        if self.aggregations:
+            dsl.aggregations = self.aggregations
```

Comment:
While small, doesn't the `=` here rather than an `extend` risk that we could clobber some aggregations if for some reason someone's managed to put aggregations in at different places?